libs/openchallenges/challenge/src/lib/challenge-contributors/challenge-contributors.component.ts

```diff
@@ -61,6 +62,12 @@ export class ChallengeContributorsComponent implements OnInit {
       );
   }
 
+  private sortOrgs(orgs: Organization[]): Organization[] {
+    return orgs.sort(
+      (a, b) => (b.challengeCount || 0) - (a.challengeCount ?? 0)
```

Comment:
Nit: on the challenge profile page, could we sort the organizations alphabetically instead?

Reply:
>Nit: on the challenge profile page, could we sort the organizations alphabetically instead?

Sure thing!